ui/src/layout/Login.js

```diff
@@ -33,6 +32,8 @@ const useStyles = makeStyles(
     card: {
       minWidth: 300,
       marginTop: '6em',
+      backgroundColor: theme.palette.background.default,
+      opacity: 0.8,
```

Comment:
I think this changes should be part of the themes, and not at the component level, or it can affect negatively other themes.